daemon/containerd/image_exporter.go

```diff
@@ -313,7 +313,8 @@ func (i *ImageService) LoadImage(ctx context.Context, inTar io.ReadCloser, outSt
 		i.LogImageEvent(img.Target.Digest.String(), img.Target.Digest.String(), events.ActionLoad)
 
 		if err != nil {
-			return errors.Wrap(err, "failed to unpack loaded image")
+			// The image failed to unpack, but is already imported, log the error but don't fail the whole load.
+			fmt.Fprintf(progress, "Error unpacking image %s: %v\n", name, err)
```

Comment:
Ah! LOL, I just commented about this on the previous commit; perhaps the previous commit is OK to be removed then?

Reply:
Ah, good point, done!